labgrid/target.py

```diff
@@ -219,6 +219,30 @@ def get_driver(self, cls, *, name=None, activate=True):
         """
         return self._get_driver(cls, name=name, activate=activate)
 
+    def get_driver_or_new(self, cls, *, name=None, activate=True, binding_map=None):
```

Comment:
As this is a special case for the client, I'd prefer to have this in the client module. For other use-cases, you either build a Target with Drivers in code or instantiate it from an Environment, but not a mix of both.

Reply:
Okay, I've moved this to the ClientSession.